osu.Game/Skinning/ISkinnableDrawable.cs

```diff
@@ -14,5 +14,11 @@ public interface ISkinnableDrawable : IDrawable
         /// Whether this component should be editable by an end user.
         /// </summary>
         bool IsEditable => true;
+
+        /// <summary>
+        /// <see langword="false"/> if this <see cref="ISkinnableDrawable"/>'s <see cref="Drawable.Anchor"/> is automatically determined by proximity,
```

Comment:
This also needs to mention that it's *only in the context of editing the layout*. Right now it sounds like it's always going to be determined on application, which is not correct.